.gitlab-ci.yml

```diff
@@ -0,0 +1,12 @@
+npmpublish:
```

Comment:
do you plan to relocate to gitlab ?

Reply:
Thanks for reading!
This PR contains only diff left after splitting the big #121
So I'll remove this ;)